specification/domainservices/resource-manager/Microsoft.AAD/stable/2020-01-01/domainservices.json

```diff
@@ -63,6 +63,49 @@
         }
       }
     },
+    "/providers/Microsoft.AAD/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.AAD/domainServices/{domainServiceName}": {
```

Comment:
This URI is not valid.  The /providers/ segment before /subscriptions/ is not a pattern that ARM supports.  You already have a "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.AAD/domainServices/{domainServiceName}" path below

Reply:
@pilor - I got a swagger completeness violation for not having this defined. Any idea what is wrong then?

https://msazure.visualstudio.com/Swagger%20KPIs/_workitems/edit/8187980

![image](https://user-images.githubusercontent.com/4138868/96647535-b09e3800-12e2-11eb-995d-cbe861b5570f.png)

Reply:
That URI is invalid and should have resulted in an error response from your RP (or not routed to your RP at all).  I don't see anything with that form in the logs in the last 24 days.  The correct path forward would be to do more validation of incoming request URIs in your RP and return a 400 or similar for URIs that are malformed

Reply:
@pilor - I am seeing the live drilldown show that this URI is being used in the last 7 days. Are you checking using different logs?

![image](https://user-images.githubusercontent.com/4138868/96658591-bfdcb000-12f9-11eb-9f38-cc2bfbdf5775.png)

![image](https://user-images.githubusercontent.com/4138868/96658610-cf5bf900-12f9-11eb-97c6-5510f1858918.png)

Reply:
@pilor - Is it possible my ARM manifest endpoint is wrong? This is the endpoint I am using: "endpointUri": "https://arm.v2.aaddc.activedirectory.windowsazure.com/providers/Microsoft.AAD/"

Reply:
Maybe.  Please work with the contracts in your incident that own the swagger correctness KPIs.  They may have a bug when RPs use endpoints like that

Reply:
@pilor - I have removed this section. I will follow up with KPI team for this part. Could you go ahead and approve this PR, so the fixes to the other correctness issues can go through. Thank you!